docs/use-cases/payments/lighting-network-like-payments/index.md

```diff
@@ -0,0 +1,26 @@
+# Lighting Network Payments
+
+> User wallet to user wallet payment network utilising trust in the large SPO.
+
+:::caution This is a legacy article
```

Comment:
I think you probably copy/pasted this so I would remove it since this is a new page not a legacy one.

Reply:
sure, I just try to keep up with existing pages format :)